channels/fast-4.4.yaml

```diff
@@ -3,3 +3,6 @@ versions:
 - 4.3.12
 - 4.3.13
 # No 4.3.14 because it was missing version which 4.3.14 can be upgraded. It is rebuilt as 4.3.15
+# No 4.3.15 as we moved to build 4.3.16 to fix https://bugzilla.redhat.com/show_bug.cgi?id=1827337
```

Comment:
It's nice to summarize the bug so folks don't have to click through.  Like:

```yaml
# No 4.3.15 because it did not clear DefaultSecurityContextConstraints_Mutated: https://bugzilla.redhat.com/show_bug.cgi?id=1827337
```

or some such.